recipes/perl-btl_bloomfilter/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+mkdir -p $PREFIX/bin
+cd swig/
+
+
+PERL5DIR=`(perl -e 'use Config; print $Config{archlibexp}, "\n";') 2>/dev/null`
+swig -Wall -c++ -perl5 BloomFilter.i
+g++ -c BloomFilter_wrap.cxx -I$PERL5DIR/CORE -fPIC -Dbool=char -O3
+g++ -Wall -shared BloomFilter_wrap.o -o BloomFilter.so -O3
+
+
+h2xs -n BTL::BloomFilter -O  -F -'I ../../../'
+cd BTL-BloomFilter
+perl Makefile.PL
+make
+make install
```

Comment:
Does this really install into $PREFIX/bin or under $PREFIX?

The tests do no catch the latter :(